packages/atomic/src/components/commerce/atomic-commerce-load-more-products/e2e/atomic-commerce-load-more-products.e2e.ts

```diff
@@ -0,0 +1,63 @@
+import {test, expect} from './fixture';
+import {AtomicCommerceLoadMoreProductsLocators} from './page-object';
+
+test.beforeEach(async ({page}) => {
+  await page.goto(
+    'http://localhost:4400/iframe.html?id=atomic-commerce-load-more-products--in-page&viewMode=story'
+  );
+});
+
+test('should be A11y compliant', async ({loadMore, makeAxeBuilder}) => {
+  await loadMore.hydrated.waitFor();
+  const accessibilityResults = await makeAxeBuilder().analyze();
+  expect(accessibilityResults.violations).toEqual([]);
+});
+
+test('should display a recap of the amount of results', async ({loadMore}) => {
+  await await expect(loadMore.summary()).toBeVisible();
+});
+
+test('should display a load more button', async ({loadMore}) => {
+  await loadMore.button.waitFor({state: 'visible'});
+  await expect(loadMore.button).toBeVisible();
+});
+
+test('should display a progress bar', async ({loadMore}) => {
+  await expect(loadMore.progressBar).toBeVisible();
+});
+
+test('should a progress value between 1 and 100', async ({loadMore}) => {
+  await expect(loadMore.progressValue).toHaveCSS('width', /^.+$/);
```

Comment:
Curious to see if https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-screenshot
could abstract the gut of the CSS and focus on the behavior